pkg/log/zap/flags.go

```diff
@@ -37,8 +37,8 @@ func init() {
 	zapFlagSet = pflag.NewFlagSet("zap", pflag.ExitOnError)
 	zapFlagSet.BoolVar(&development, "zap-devel", false, "Enable zap development mode (changes defaults to console encoder, debug log level, and disables sampling)")
 	zapFlagSet.Var(&encoderVal, "zap-encoder", "Zap log encoding ('json' or 'console')")
-	zapFlagSet.Var(&levelVal, "zap-level", "Zap log level (one of 'debug', 'info', 'error')")
-	zapFlagSet.Var(&sampleVal, "zap-sample", "Enable zap log sampling")
+	zapFlagSet.Var(&levelVal, "zap-level", "Zap log level (one of 'debug', 'info', 'error' or any integer value > 0)")
+	zapFlagSet.Var(&sampleVal, "zap-sample", "Enable zap log sampling. Sampling will be disabled for log levels > 0")
```

Comment:
To match our [check in `getConfig`](https://github.com/operator-framework/operator-sdk/pull/1210/files#diff-4407315726717f30063d558928ad9188R86), I'd suggest this change to the wording:
```suggestion
	zapFlagSet.Var(&sampleVal, "zap-sample", "Enable zap log sampling. Sampling will be disabled for integer log levels > 1")
```

Reply:
Changed it.
```
[jesusr@transam foo-operator{master}]$ operator-sdk up local --help
The operator-sdk up local command launches the operator on the local machine
by building the operator binary with the ability to access a
kubernetes cluster using a kubeconfig file.

Usage:
  operator-sdk up local [flags]

Flags:
      --go-ldflags string           Set Go linker options
  -h, --help                        help for local
      --inject-owner-ref            (ansible operator) The ansible operator will inject owner references unless this flag is false (default true)
      --kubeconfig string           The file path to kubernetes configuration file; defaults to location specified by $KUBECONFIG with a fallback to $HOME/.kube/config if not set
      --max-workers int             (ansible operator) Maximum number of workers to use. Overridden by environment variable. (default 1)
      --namespace string            The namespace where the operator watches for changes.
      --operator-flags string       The flags that the operator needs. Example: "--flag1 value1 --flag2=value2"
      --reconcile-period duration   (ansible operator) Default reconcile period for controllers (default 1m0s)
      --watches-file string         (ansible operator) Path to the watches file to use (default "./watches.yaml")
      --zap-devel                   Enable zap development mode (changes defaults to console encoder, debug log level, and disables sampling)
      --zap-encoder encoder         Zap log encoding ('json' or 'console')
      --zap-level level             Zap log level (one of 'debug', 'info', 'error' or any integer value > 1) (default info)
      --zap-sample sample           Enable zap log sampling. Sampling will be disabled for integer log levels > 1
```